README.md

```diff
@@ -298,6 +300,22 @@ If no `error` is supplied, the `response` value can be any plain JavaScript
 value sent by the server, including any values that can be unserialized from
 [JSON].
 
+<a name="session.listen" />
+
+---
+
+> *`void`* [**`session.listen`**](#session.listen) `(...namespaces)`
+
+Listens to notifications on the given `namespaces` variable argument list.
```

Comment:
We probably want to include info about *why* you need to call listen in the first place, and talk about how it is safe to call listen with a namespace you're already listening to etc. etc.

The "Managed module usage" section of the docs also makes use of notifications, and we probably need to update that to show that you have to listen to the appropriate namespace as well.